inc/saitypes.h

```diff
@@ -1428,6 +1428,21 @@ typedef enum _sai_stats_mode_t
      * @brief Read and clear after reading
      */
     SAI_STATS_MODE_READ_AND_CLEAR = 1 << 1,
+
+    /**
+     * @brief Build read statistics
```

Comment:
nit: bulk (here and below)

Reply:
Fixed
https://github.com/opencomputeproject/SAI/pull/1352/commits/9896eb8fd6e18cc3a861dd37b16c0d14c8619bda